core/src/main/java/hudson/model/TextParameterDefinition.java

```diff
@@ -37,7 +38,7 @@ public TextParameterDefinition(String name, String defaultValue, String descript
         super(name, defaultValue, description);
     }
 
-    @Extension
+    @Extension @Symbol({"text","textParam"})
```

Comment:
OK…but we should have some sort of consistency I suppose (see `BooleanParameterDefinition`).